src/include/migraphx/operation.hpp

```diff
@@ -278,11 +347,10 @@ void from_value_op(T& x, const value& v)
  *      std::ptrdiff_t output_alias(const std::vector<shape>& input) const;
  *      void finalize(context& ctx,const shape& output,const std::vector<shape>& input) ;
  *      shape compute_shape(const std::vector<shape>& input) const;
- *      argument compute(context& ctx,const shape& output,const std::vector<argument>& input) const;
- *      argument compute(const shape& output,const std::vector<argument>& input) const;
- *      value to_value() const;
- *      void from_value(const value& v) ;
- *      value attributes() const;
+ *      shape compute_shape(const std::vector<shape>& inputs,const std::vector<module_ref>&
```

Comment:
was this auto-generated? If not, it looks like we should probably fix the formatting so each new function is on a new line.

Reply:
Yes, it is auto-generated. I did not know why it is like this.